docs/pipelines/languages/dotnet-core.md

```diff
@@ -240,12 +240,26 @@ You build your .NET Core project by running `dotnet build` command in your pipel
 
 ::: moniker range="vsts"
 
-To build your project using .NET Core task, add the following snippet to your `azure-pipelines.yml` file:
+To build your project, add the following `dotnet build` snippet to your `azure-pipelines.yml` file:
 
 ```yaml
 - script: dotnet build # Include additional options such as --configuration to meet your need
 ```
 
+Or, to build using the **.NET Core** task:
+
+```yaml
+# .NET Core
+# Restore NuGet packages.
+- task: DotNetCoreCLI@2
+  inputs:
+    command: 'build'
+    projects: '**/*.csproj'
+    #verbosityRestore: 'detailed' # Options: quiet, minimal, normal, detailed, diagnostic
+```
+
+For more information about the .NET Core task, see [Build: .NET Core CLI](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/build/dotnet-core-cli?view=vsts).
```

Comment:
@davetheunissen could you please remove `?view=vsts` from the hyperlink?  Then it will show the .NET Core task docs for a specific version of TFS if someone defaulted to one previously.